v1/remote/transport/bearer.go

```diff
@@ -22,15 +22,17 @@ import (
 	"net/url"
 
 	"github.com/google/go-containerregistry/authn"
+	"github.com/google/go-containerregistry/name"
 )
 
 type bearerTransport struct {
-	inner   http.RoundTripper
-	basic   authn.Authenticator
-	bearer  *authn.Bearer
-	realm   string
-	service string
-	scope   string
+	inner    http.RoundTripper
+	basic    authn.Authenticator
+	bearer   *authn.Bearer
+	registry name.Registry
+	realm    string
+	service  string
+	scope    string
```

Comment:
Looking at this list, it's long.  Can we add comments to each of these things describing their purpose?